<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advance</class>
 <widget class="QDialog" name="advance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>advance</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QLineEdit" name="textBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(243, 243, 243);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnClear">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 127, 168);</string>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnPower">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 127, 168);</string>
     </property>
     <property name="text">
      <string>^2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnPourcent">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 127, 168);</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnDiv">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 127, 168);</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btnSin">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 90, 219);</string>
     </property>
     <property name="text">
      <string>Sin</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn1">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btn2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btn3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="btnMul">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 127, 168);</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="btnCos">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 90, 219);</string>
     </property>
     <property name="text">
      <string>Cos</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btn6">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnSub">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 127, 168);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="btnTan">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 90, 219);</string>
     </property>
     <property name="text">
      <string>Tan</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btn7">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btn8">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btn9">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnSum">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 127, 168);</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="btnCot">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 90, 219);</string>
     </property>
     <property name="text">
      <string>Cot</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="btn0">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="btnNeg">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="btnPoint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 255, 60);</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="btnEqual">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 226, 52);</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="btnSqrt">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 90, 219);</string>
     </property>
     <property name="text">
      <string>âˆš</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
